readthedocs/doc_builder/python_environments.py

```diff
@@ -0,0 +1,247 @@
+            # virtualenv.  Using the -I option makes sure it gets installed
+            # even if it is already installed system-wide (and
+            # --system-site-packages is used)
+            cmd.append('-I')
+        cmd.extend(requirements)
+        self.build_env.run(
+            *cmd,
+            bin_path=self.venv_bin(version=self.version.slug)
+        )
+
+    def install_user_requirements(self):
+        requirements_file_path = self.config.requirements_file
+        if not requirements_file_path:
+            builder_class = get_builder_class(self.project.documentation_type)
+            docs_dir = (builder_class(build_env=self.build_env, python_env=self)
+                        .docs_dir())
+            for path in [docs_dir, '']:
+                for req_file in ['pip_requirements.txt', 'requirements.txt']:
+                    test_path = os.path.join(self.checkout_path, path, req_file)
+                    if os.path.exists(test_path):
+                        requirements_file_path = test_path
+                        break
+
+        if requirements_file_path:
+            self.build_env.run(
+                'python',
+                self.venv_bin(version=self.version.slug, filename='pip'),
+                'install',
+                '--exists-action=w',
+                '--cache-dir',
+                self.project.pip_cache_path,
+                '-r{0}'.format(requirements_file_path),
+                cwd=self.checkout_path,
+                bin_path=self.venv_bin(version=self.version.slug)
+            )
+
+
+class Conda(PythonEnvironment):
+
+    def venv_path(self, version=LATEST):
+        return os.path.join(self.project.doc_path, 'conda', version)
+
+    def setup_base(self):
+        conda_env_path = os.path.join(self.project.doc_path, 'conda')
+        version_path = os.path.join(conda_env_path, self.version.slug)
+
+        if os.path.exists(version_path):
+            # Re-create conda directory each time to keep fresh state
+            self._log('Removing existing conda directory')
+            shutil.rmtree(version_path)
+        self.build_env.run(
+            'conda',
+            'create',
+            '--yes',
+            '--name',
+            self.version.slug,
+            'python={python_version}'.format(python_version=self.config.python_version),
+            environment={'CONDA_ENVS_PATH': conda_env_path}
+        )
+
+    def install_core_requirements(self):
+        conda_env_path = os.path.join(self.project.doc_path, 'conda')
+
+        # Use conda for requirements it packages
+        requirements = [
+            'sphinx==1.3.1',
+            'Pygments==2.0.2',
+            'docutils==0.11',
+            # 'mkdocs==0.14.0',
+            'mock==1.0.1',
+            'pillow==3.0.0',
+            # 'readthedocs-sphinx-ext==0.5.4',
+            'sphinx_rtd_theme==0.1.7',
+            'alabaster>=0.7,<0.8,!=0.7.5',
+            # 'recommonmark==0.1.1',
```

Comment:
Can these commented packages be removed here? Ditto for the `Virtualenv` class for consistency